Dockerfile.test

```diff
@@ -0,0 +1,6 @@
+FROM localhost/coreos-override-builder as builder
+COPY test /root
```

Comment:
I think this needs to be `COPY test /root/test` to maintain the `test` directory. Otherwise it'll copy the insides of `test` into `/root`.

Reply:
Yeah, I reworked things to require destdir at the last minute before pushing the PR and after testing.  It's cleaner if we require it to not exist and to populate it.